src/main/java/uk/gov/companieshouse/company/profile/model/CompanyProfileDocument.java

```diff
@@ -1,22 +1,28 @@
 package uk.gov.companieshouse.company.profile.model;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import org.bson.types.ObjectId;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
-import uk.gov.companieshouse.api.company.CompanyProfile;
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.gov.companieshouse.api.company.Data;
 
 @Document(collection = "company_profile")
```

Comment:
Please externalise this collection name in yml file as some talks going on to do parallel testing with temp collection name initially

Reply:
Done